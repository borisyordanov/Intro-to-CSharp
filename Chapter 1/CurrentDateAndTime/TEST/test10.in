2991
11
2
5
49
3

