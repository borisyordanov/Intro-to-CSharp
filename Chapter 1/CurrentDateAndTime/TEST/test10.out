11/2/2991 5:49:03 AM
